### ISSUE TITLE
Newly-created Trello cards having tags may lose their labels

### ISSUE THREAD

**Issue by coddingtonbear** [coddingtonbear/inthe.am#321]
This *probably* happens because synchronization has not yet completed, so the operation is something like:

```
@startuml
Trello -> IntheAM: New card created (without tags)
Trello -> IntheAM: New label added
IntheAM -> Trello: (in parallel with the above): Synchronize task (without tags) with trello card
Trello -> IntheAM: Label removed
@enduml
```
